### PR TITLE
Transfer cleanup

### PR DESCRIPTION
**Pull request by madMatchstick**
cleanup from repo transfer - mostly mds

### CODE REVIEW

**Review by ZacharyWills — COMMENTED** [review]
Add repo description (some hybrid of the README.md new lines 3/5 with regards to keeping the TOPMODEL and adding BMI functionality). 

Looks good outside of that. 👍

**Comment by madMatchstick** [comment]
> Add repo description (some hybrid of the README.md new lines 3/5 with regards to keeping the TOPMODEL and adding BMI functionality).
> 
> Looks good outside of that. 👍

Done ty. @ZacharyWills - are you able to approve/review?

File: .gitignore [hunk 1]
```diff
@@ -75,3 +75,20 @@ bower_components/
...
+run_bmi
+z_trash
+test/hyd.out
+test/topmod.out
+test/run_bmi_test
+test/z_trash_test
+
+# Misc #
+########
+make_and_run_bmi.sh
```
Comment (robertbartel) [inline .gitignore:94]: I suspect the file _make_and_run_bmi.sh_ needs to be removed from the repo, based on _INSTALL.md_.  That or this line in _.gitignore_ should probably be removed. 

 If it was intended to have the file in the repo _and_ have it be ignored, I think an explicit comment should be added to explain things.

File: README.md [hunk 2]
```diff
@@ -1,101 +1,5 @@
...
-  This C version by Fred Ogden, Sept. 2009.  Converted to ANSI C
-  and given somewhat more meaningful variable names.  Compared
-  against original FORTRAN version on a simple data set with no
-  infiltration excess.
-  
-  after Keith Beven 1985
-  Revised for distribution 1993,1995
-
-****************************************************************
-  This program is distributed freely with only two conditions.
```
Comment (robertbartel) [inline README.md:17]: I presume these conditions for use are no longer applicable?

File: README.md [hunk 3]
```diff
@@ -1,101 +1,5 @@
...
-  SBAR at end of time step
-
-  Current program limits are:
-          Number of time steps = 2500
-          Number of subcatchments = 10
-          Number of ln(a/tanB) increments = 30
-          Number of subcatchment routing ordinates = 10
-          Number of time delay histogram ordinates = 20
-          Size of subcatchment pixel maps = 100 x 100
-
```
Comment (robertbartel) [inline README.md:85]: There are this item and a few other similar things being removed from this file that seem like they could still be important (in addition to the usage conditions, which I commented on above).  Is this all either no longer applicable or documented elsewhere?

**Review by robertbartel — CHANGES_REQUESTED** [review]
There is one thing related to a file and _.gitignore_ that needs to be addressed (see my in-line comment).

I also put in a few comments on things being removed from the main _README.md_.  I later found where they still are in the original Fortran file, so I'm guessing removing them is likely are fine, but I'll leave the note just in case.

**Comment by madMatchstick** [comment]
> There is one thing related to a file and _.gitignore_ that needs to be addressed (see my in-line comment).
> 
> I also put in a few comments on things being removed from the main _README.md_. I later found where they still are in the original Fortran file, so I'm guessing removing them is likely are fine, but I'll leave the note just in case.

This is correct. The previous README was basically a c/p from source code opening comments (both FORTRAN and C), so think we're okay here.

File: .gitignore (hunk 1 above)
Comment (madMatchstick) [reply .gitignore:94]: Now using Make to build/run so I'll just remove shell script completely. Good catch.

File: README.md (hunk 2 above)
Comment (madMatchstick) [reply README.md:17]: No longer in README but original comment block still in [topmodel.c](https://github.com/NOAA-OWP/topmodel/blob/master/src/topmodel.c#L12).  Best to leave in source FORTRAN and C code but I'll ask about this.

File: README.md (hunk 3 above)
Comment (madMatchstick) [reply README.md:85]: Again, still stated in [topmod.c](https://github.com/NOAA-OWP/topmodel/blob/b05a89430525161a5d086fdfb480e434c417f428/src/topmodel.c#L73). 
fyi - don't believe max Number of time steps = 2500 is ever enforced; will confirm & comment accordingly.  Other limits here, however, are set in bmi initialize()

**Review by robertbartel — APPROVED** [review]

**Comment by ZacharyWills** [comment]
Good job everyone!
🥇

**Pull request merged**